aten/src/ATen/native/cuda/BinaryRemainderKernel.cu

```diff
@@ -20,6 +26,18 @@ void remainder_kernel_cuda(TensorIterator& iter) {
         return r;
       });
     });
+  }
+  else if (isComplexType(iter.dtype())) {
+    AT_DISPATCH_COMPLEX_TYPES(iter.dtype(), "remainder_cuda", [&]() {
+      gpu_kernel_with_scalars(iter,
+        []GPU_LAMBDA(scalar_t a, scalar_t b) -> scalar_t {
+          scalar_t mod = a - floor_wrapper (a / b) * b;
+          const scalar_t zero = scalar_t(0); 
+          if ((mod != zero) && ((std::real(b) < std::real(zero)) != (std::real(mod) < std::real(zero)))) 
+            mod += b;
```

Comment:
From the definition of reminder it must be `|mod| < |b|`. So, it is worth testing this as well.

Reply:
Also, I am also not sure this is the right way for complex, I am thinking about the following algorithm:
```
r_len = reminder(|a| / |b|)
r_arg = (a - r_len * b) / (a - r_len * b).abs()
return r_len * r_arg
```